src/main.rs

```diff
@@ -78,52 +78,27 @@ where
 }
 
 #[cfg(not(target_arch = "wasm32"))]
-static LOG_TERM_SENDER: once_cell::sync::OnceCell<luminol_term::TermSender> =
+static LOG_BYTE_SENDER: once_cell::sync::OnceCell<std::sync::mpsc::Sender<u8>> =
     once_cell::sync::OnceCell::new();
-
-#[cfg(not(target_arch = "wasm32"))]
-static LOG_BYTE_SENDER: once_cell::sync::OnceCell<luminol_term::ByteSender> =
-    once_cell::sync::OnceCell::new();
-
 #[cfg(not(target_arch = "wasm32"))]
 static CONTEXT: once_cell::sync::OnceCell<egui::Context> = once_cell::sync::OnceCell::new();
 
-#[cfg(not(target_arch = "wasm32"))]
 /// A writer that writes to Luminol's log window.
-struct LogWriter(luminol_term::termwiz::escape::parser::Parser);
+#[cfg(not(target_arch = "wasm32"))]
+struct LogWriter;
 
 #[cfg(not(target_arch = "wasm32"))]
-impl Write for LogWriter {
+impl std::io::Write for LogWriter {
     fn write(&mut self, buf: &[u8]) -> std::io::Result<usize> {
-        LOG_BYTE_SENDER
-            .get()
-            .unwrap()
-            .try_send(buf.into())
-            .map_err(std::io::Error::other)?;
-
-        let parsed = self.0.parse_as_vec(buf);
-
-        // Convert from LF line endings to CRLF so that wezterm will display them properly
-        let mut vec = Vec::with_capacity(2 * parsed.len());
-        for action in parsed {
-            if action
-                == luminol_term::termwiz::escape::Action::Control(
-                    luminol_term::termwiz::escape::ControlCode::LineFeed,
-                )
-            {
-                vec.push(luminol_term::termwiz::escape::Action::Control(
-                    luminol_term::termwiz::escape::ControlCode::CarriageReturn,
-                ));
+        let sender = LOG_BYTE_SENDER.get().unwrap();
+        for &byte in buf {
+            #[cfg(target_os = "linux")]
+            if byte == b'\n' {
+                let _ = sender.send(b'\r');
```

Comment:
This newline substitution needs to be performed on all platforms since logs always use LF newlines and the terminal behaves the same on all platforms